packages/experience/src/pages/IdentifierSingleSignOn/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { AgreeToTermsPolicy, experience } from '@logto/schemas';
+import { useTranslation } from 'react-i18next';
+
+import IdentifierPageLayout from '@/Layout/IdentifierPageLayout';
+import useTerms from '@/hooks/use-terms';
+
+import SingleSignOnForm from '../SingleSignOnEmail/SingleSignOnForm';
```

Comment:
should not directly import components from other pages?  Extract to a shared place instead.

Reply:
Good call, will extract related component to a shared place together with the identifier. sign-in / register components in another PR.